### ISSUE TITLE
Germline bias dictates cross-serotype reactivity in a common dengue-virus-specific CD8+ T cell response

### ISSUE THREAD

**Issue by mikessh** [antigenomics/vdjdb-db#224]
http://www.nature.com/ni/journal/vaop/ncurrent/full/ni.3850.html

**Comment by RenskeVroomans** [comment]
I received the sequence data from David Price a few days before. I am almost ready to submit them, I just have a few questions (also for him). They have patients with a secondary dengue infection from a different serotype. They doublestain the cells from these patients with two tetramers (one for the old and one for the new infection). So some of the T cells bind two tetramers at the same time. I filled these sequences in twice, one for each epitope they bind, and under meta.cell.subset I added dp (double-positive) behind the CD8+ marker to indicate this. 
So I would like to know if the indication is necessary, or whether there is a more appropriate field to indicate this. Also, the frequency of these cells is now also copied, i.e. the same for both entries; this may not be appropriate but there is no info on whether the t cell mostly binds one or the other.
